client/src/pages/Chat.tsx

```diff
@@ -54,7 +55,7 @@ export const Chat = () => {
     if (!chatLoading && !chat) navigate('/selection');
   }, [chat, chatLoading, navigate]);
 
-  useEffect(() => {
+  useEffect(() => { // todo: getName from chatObj instead?
```

Comment:
Regarding to line 62:
`      const compName = await getNameById(chat![getOppositeRoleFieldName(userRole)]!);
`
maybe we should add another field to the chatSchema and then use that instead of requesting it from the server every time

`const chatSchema = z.object({
  id: z.string(),
  supporterId: z.string().nullable(),
  supporteeId: z.string().nullable(),
  createdAt: z.instanceof(Timestamp),
  status: statusEnum.default('active'),
  supporteeName: z.string().nullable(),
**supporterName: z.string().nullable()**
});`